core/test/oracle/Voting.js

```diff
@@ -1310,9 +1321,18 @@ contract("Voting", function(accounts) {
       );
     });
 
-    // RewardsRetrieved event gets emitted for every reward retrieval that's attempted, even if no tokens are minted.
+    // RewardsRetrieved event gets emitted for every reward retrieval that's attempted, even if no tokens are minted
```

Comment:
Should this comment be "RewardsRetroeved event does not get emitted if not tokens are minted because the vote was wrong"?

Reply:
The event *does* get minted in the case being tested here.